app/Community/Commands/ConvertGameShortcodesToHubs.php

```diff
@@ -0,0 +1,267 @@
+                        ->where('id', $forumTopicComment->id)
+                        ->update([
+                            'body' => $updatedPayload,
+                            'updated_at' => DB::raw('updated_at'), // don't change the value
+                        ]);
+                }
+            }
+
+            $progressBar->advance(count($forumTopicComments));
+        });
+        $progressBar->finish();
+
+        $this->info("\nForumTopicComments migration completed successfully.");
+    }
+
+    private function migrateMessageShortcodes(): void
+    {
+        // Get the total count of messages that need to be processed.
+        $totalMessages = Message::where('body', 'like', '%[game=%')->count();
+
+        $progressBar = $this->output->createProgressBar($totalMessages);
+        Message::where('body', 'like', '%[game=%')->chunkById(1000, function ($messages) use ($progressBar) {
+            // Collect all game IDs to fetch hub mappings in bulk.
+            $gameIds = [];
+
+            /** @var Message $message */
+            foreach ($messages as $message) {
+                preg_match_all('/\[game=(\d+)\]/', $message->body, $matches);
+                $gameIds = array_merge($gameIds, $matches[1]);
+            }
+
+            // Remove duplicates and fetch games that are legacy hubs.
+            $gameIds = array_unique($gameIds);
+            $legacyHubs = Game::whereIn('ID', $gameIds)
+                ->where('ConsoleID', 100)
+                ->get()
+                ->keyBy('ID');
+
+            // Fetch corresponding hub IDs for these legacy games.
+            $hubMappings = GameSet::where('type', GameSetType::Hub)
+                ->whereIn('game_id', $legacyHubs->pluck('ID'))
+                ->get()
+                ->keyBy('game_id');
+
+            // Process each message.
+            /** @var Message $message */
+            foreach ($messages as $message) {
+                $originalBody = $message->body;
+                $updatedBody = preg_replace_callback('/\[game=(\d+)\]/', function ($matches) use ($legacyHubs, $hubMappings) {
+                    $gameId = (int) $matches[1];
+
+                    // Only replace if it's a legacy hub and we have a mapping.
+                    if ($legacyHubs->has($gameId) && $hubMappings->has($gameId)) {
+                        return "[hub={$hubMappings->get($gameId)->id}]";
+                    }
+
+                    return $matches[0];
+                }, $message->body);
+
+                // Update the message in the DB only if it has actually changed.
+                // Use `DB` so we don't change the `updated_at` value.
+                if ($originalBody !== $updatedBody) {
+                    DB::table('messages')
+                        ->where('id', $message->id)
+                        ->update([
+                            'body' => $updatedBody,
+                            'updated_at' => DB::raw('updated_at'), // don't change the value
```

Comment:
```
  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'updated_at' in 'SET' (Connection: mysql, SQL: update `messages` set `body` = ... , `updated_at` = updated_at where `id` = 19)
```
Is there another PR that adds/removes this column and I'm just out of sync?

Reply:
Also, why use a direct query to update a single record?
```
$message->body = $updatedBody;
$message->timestamps = false;
$message->save();
```
(same comment for forums topic comments above)

Reply:
> Is there another PR that adds/removes this column and I'm just out of sync?

Nope, this is a bad copy+paste job after fixing a bug in the command.

> Also, why use a direct query to update a single record?

I have no idea why I landed on this solution anymore. I tried `->timestamps` and for some reason, found on my local it wasn't working. Now it is. I must have had a logic error somewhere.

Should all be fixed up now in latest (along with the merge conflict resolved).